app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -243,6 +243,18 @@ def cfi_checkbox_was_selected?
 
   private
 
+  def log_error(error)
+    Rails.logger.error(
+      "Form526ClaimsFastTrackingConcern #{id} encountered an error: #{error.message}",
+      submission_id: id,
+      error_message: error.message
```

Comment:
Will these error messages contain any PII?

Reply:
Resolved in e7cbc09de1